### PR TITLE
Cleaning useless colon

### PR DESCRIPTION
**Pull request by senayar**
Remove useless colon

### CODE REVIEW

**Comment by danmcp** [comment]
@jwforres Review please

File: assets/app/scripts/filters/date.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ angular.module('openshiftConsole')
       if (!timestampLhs) {
         return timestampLhs;
       }
-      timestampRhs = timestampRhs || Date();
+      timestampRhs = timestampRhs || new Date();
```
Comment (liggitt) [inline assets/app/scripts/filters/date.js]: this seems questionable
http://stackoverflow.com/a/9584767/54696

Comment (liggitt) [inline assets/app/scripts/filters/date.js]: I think the existing behavior (string, not object) may have been intentional

**Comment by liggitt** [comment]
need to run hack/build-assets.sh and check in resulting bindata.go changes

**Comment by jwforres** [comment]
@senayar please rebase and run hack/build-assets.sh and check in the bindata.go file

**Comment by danmcp** [comment]
@jwforres @senayar Bump

**Comment by jwforres** [comment]
[merge]

@liggitt fyi there is no change to bindata because the minification scripts must already strip trailing commas

**Comment by senayar** [comment]
@jwforres I rebased the branch I tried to run hack/build-assets.sh but it raise me an error : 

`hack/build-assets.sh: line 18: Godeps/_workspace/bin/go-bindata: No such file or directory`

I think I am missing some dependencies to run it.

**Pull request merged**